packages/next-mdx/readme.md

```diff
@@ -51,7 +51,7 @@ module.exports = withMDX({
 })
 ```
 
-Optionally you can match other file extensions for MDX compilation, by default only `.mdx` is supported
+Optionally you can match other file extensions for MDX compilation, as by default only `.mdx` is supported:
```

Comment:
```suggestion
By default MDX will only match and compile MDX files with the `.mdx` extension.
However, it can also be optionally configured to handle markdown files with the `.md` extension, as shown below:

```